examples/silicon.jl

```diff
@@ -9,6 +9,6 @@ Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
 atoms     = [Si, Si]
 positions = [ones(3)/8, -ones(3)/8]
 
-model = model_LDA(lattice, atoms, positions)
+model = DFTK.model_PBE0(lattice, atoms, positions)
```

Comment:
I would not commit that here. Better to make a new example (e.g. one where you restrict to the exact case that can be covered by the code.